microcosm_flask/paging.py

```diff
@@ -35,9 +35,11 @@ class PaginatedListSchema(Schema):
 
 
 class Page(object):
-    def __init__(self, offset, limit):
+
+    def __init__(self, offset, limit, **rest):
```

Comment:
I've generalized the `Page` object to retain "other" arguments from its pagination. I have not yet generalized it's `from_query_string` function but I think I should. Feedback here please.